crispy_tailwind/templates/tailwind/layout/select.html

```diff
@@ -0,0 +1,15 @@
+{% load crispy_forms_filters %}
+{% load l10n %}
+
+<div class="relative">
+<select class="{% if field.errors %}border border-red-500 {% endif %}block appearance-none w-full bg-gray-200 border border-gray-200 text-gray-700 py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-gray-500" name="{{ field.name }}" {{ field.field.widget.attrs|flatatt }}>
+    {% for value, label in field.field.choices %}
+        {% include "tailwind/layout/select_option.html" with value=value label=label %}
+    {% endfor %}
+</select>
+<div class="pointer-events-none absolute inset-y-0 right-0 flex items-center px-2 text-gray-700">
+    <svg class="fill-current h-4 w-4" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z"/></svg>
+  </div>
+</div>
+{% include 'tailwind/layout/help_text.html' %} 
+</div>
```

Comment:
As per previous comment - I think my error has also resulted in this additional closing `div` here.